src/sagemaker_mxnet_serving_container/handler_service.py

```diff
@@ -44,7 +44,10 @@ def __init__(self):
 
     @staticmethod
     def _user_module_transformer(model_dir=environment.model_dir):
-        user_module = importlib.import_module(environment.Environment().module_name)
+        module_name = environment.Environment().module_name
+        inference_script = model_dir + '/code' + '/{}.py'.format(module_name)
+        spec = importlib.util.spec_from_file_location(module_name, inference_script)
+        user_module = importlib.util.module_from_spec(spec)
```

Comment:
we can add some checks making sure inference_script exist before loading the module.

Reply:
Added some checks